src/app/clusters/diagnostic-logs-server/DiagnosticLogsProviderDelegate.h

```diff
@@ -59,9 +59,10 @@ class DiagnosticLogsProviderDelegate
      * returned from StartLogCollection.
      *
      * @param[in] sessionHandle  The unique handle for this log session returned from a call to StartLogCollection.
+     * @param[in] error A CHIP_ERROR value that indicates the reason for ending the log collection.
      *
      */
-    virtual CHIP_ERROR EndLogCollection(LogSessionHandle sessionHandle) = 0;
+    virtual CHIP_ERROR EndLogCollection(LogSessionHandle sessionHandle, CHIP_ERROR error) = 0;
```

Comment:
This is a backwards-incompatible change, right?  Can we do this backwards-compatibly?